coco/metric.py

```diff
@@ -48,3 +52,34 @@ def start_metrics_server(port, callbacks=None, addr=""):
     t = threading.Thread(target=httpd.serve_forever)
     t.daemon = True
     t.start()
+
+
+async def get(name, port, host="localhost"):
+    """
+    Get the value of a metric by requesting it from the prometheus web server.
+
+    Parameters
+    ----------
+    name : str
+        Name of the metric
+    port : int
+        Port of the prometheus server
+    host : str
+        Host running the prometheus server (default "localhost")
+
+    Returns
+    -------
+    Value of the metric
+    """
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"http://{host}:{port}") as resp:
+            resp.raise_for_status()
+            metrics = await resp.text()
+    for family in text_string_to_metric_families(metrics):
+        if family.name == name:
+            assert len(list(family.samples)) == 1
```

Comment:
Why is it important that their is only 1 sample?

Reply:
I wouldn't know why there would be more than 1...

Reply:
Do you?

Reply:
Why did they call the attribute `samples` .-.